extmod/libmetal/metal/config.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Can we also make this file a simple one-liner that includes `metal/micropython.h`?  Then all the configuration is in one file.

Another reason for this is so we don't have non-MIT code in this `extmod` directory.  All code currently in `extmod` is first-party code and MIT licensed.  We should try to keep it that way.

Reply:
We can just move everything to `extmod/libmetal/metal/config.h`, see updated files.

Reply:
Great!